onnxruntime/test/shared_lib/test_inference.cc

```diff
@@ -1207,7 +1207,7 @@ TEST(CApiTest, RegisterCustomOpForCPUAndCUDA) {
 }
 #endif
 
-#if (!defined(ORT_MINIMAL_BUILD)) || defined(ORT_MINIMAL_BUILD_CUSTOM_OPS)
+#if (!defined(__ANDROID__)) && ((!defined(ORT_MINIMAL_BUILD)) || defined(ORT_MINIMAL_BUILD_CUSTOM_OPS))
```

Comment:
a comment explaining why might be useful.

would it be possible to copy over the needed library and run the test on the emulator instead?
e.g., like we do here:
https://github.com/microsoft/onnxruntime/blob/a3570eb5bf98d55a85130545bc598856d0f055ae/tools/ci_build/build.py#L1627-L1630